src/catalog/table_catalog.cpp

```diff
@@ -489,7 +489,7 @@ std::shared_ptr<TableCatalogObject> TableCatalog::GetTableObject(
   auto database_object = txn->catalog_cache.GetDatabaseObject(database_oid);
   if (database_object) {
     auto table_object =
-        database_object->GetTableObject(table_name, schema_name, true);
+        database_object->GetTableObject(table_name, schema_name, std::string(), true);
```

Comment:
If I understand correctly, when schema_name is empty, the session namespace would be used to generate the temporary namespace. If you create an empty string, this may cause problems in GetTableObject.